lhotse/recipes/utils.py

```diff
@@ -0,0 +1,28 @@
+from collections import defaultdict
+from typing import Dict, Optional, Sequence, Union
+
+from lhotse.audio import RecordingSet
+from lhotse.manipulation import load_manifest
+from lhotse.supervision import SupervisionSet
+from lhotse.utils import Pathlike
+
+
+def read_manifests_if_cached(
+        dataset_parts: Optional[Sequence[str]],
+        output_dir: Optional[Pathlike]
+) -> Optional[Dict[str, Dict[str, Union[RecordingSet, SupervisionSet]]]]:
+    """Loads manifests from the disk if all of them exist in the specified paths.
+    This function is intended to speedup data preparation if it is already done before.
```

Comment:
Could you mention in the doc that the manifests are searched for using the pattern `output_dir / f'{manifest}_{part}.json'`, where `manifest` is one of `["recordings", "supervisions"]`?